docs/hydra/guides/migrate-v2.mdx

```diff
@@ -0,0 +1,110 @@
+---
+id: migrate-v2
+title: Migrate from Hydra v1 to v2
+---
+
+Follow this guide when upgrading Ory Hydra to v2. This guide covers the essential steps that are likely to be universal. If your
+deployment requires different steps, deviate from this guide at your discretion.
+
+Should you run into problems with the upgrade, please visit the community [chat](https://slack.ory.sh/) or start a
+[discussion](https://github.com/ory/hydra/discussions).
+
+:::warning
+
+Back up your data! Applying upgrades can lead to data loss if handled incorrectly.
+
+:::
+
+:::warning
+
+This is an offline migration. You will need to take Hydra offline for the duration of the migration.
+
+:::
+
+:::warning
+
+There is no down migration from v2 to v1.
+
+:::
+
+## Upgrade
+
+1. Read the release notes
+
+We try to make our [release notes](https://github.com/ory/hydra/releases) as comprehensive as possible, so make sure to give them
+a good read. They might contain information that is specific to your deployment.
+
+2. Update your dependencies
+
+Make sure you update all of your [Hydra SDK](../sdk) dependencies and [Ory Hydra itself](../install). We try to make sure that new
+versions are as backward compatible as possible, but v2 does come with breaking changes.
+
+## Migrate: Dry Run
+
+If you encounter any issues, please contact us or contribute to this guide by adding them.
+
+1. Check out [Known issues, limitations, and sanity checks](#known-issues-limitations-and-sanity-checks).
+
+2. Backup your data.
+
+3. Deploy a dry-run environment with the backed-up data.
+
+4. Take the dry-run environment offline.
+
+5. Apply the Hydra v2 patch to the dry-run environment.
+
+6. Run [`hydra migrate sql`](../cli/hydra-migrate-sql.md) to run the SQL migrations.
+
+7. Start the dry-run environment.
+
+8. Test the dry-run environment.  
+   Make sure to test your deployment before you go live. Hydra releases are tested, but each deployment is different and it is
+   critical to have your own tests.
+
+## Migrate: Go Live
+
+1. Do a dry run as described above.
+
+2. Plan your downtime.  
+   The dry run should give you an idea of how long the migrations will take.
+
+3. Take your deployment offline.
+
+4. Backup your data.  
+   Be prepared to go back online with the original database and Ory Hydra v1.
+
+5. Switch your deployment to the version successfully tested in the dry run.
+
+6. Run [`hydra migrate sql`](../cli/hydra-migrate-sql.md) to apply the SQL migrations.
+
+7. Run the [sanity checks](#known-issues-limitations-and-sanity-checks).
```

Comment:
There is a single "sanity check". The section you're referring to contains a single simple query and it could be made an integral part of the instruction.

Reply:
This section is designed to encourage people to contribute their own checks. The language is deliberately not meant to sound authoritative in order to encourage people to contribute.